quetz/tasks/mirror.py

```diff
@@ -410,7 +410,7 @@ def synchronize_packages(
     if subdirs is None:
         subdirs = KNOWN_SUBDIRS
 
-    for arch in subdirs:
+    for arch in ['noarch']:
```

Comment:
??? I think this causes the CI to fail

Reply:
I was using this branch to deploy and test on the server, sorry. Wasn't really ready for review